app/models/agendaitem.js

```diff
@@ -109,14 +108,9 @@ export default ModelWithModifier.extend({
     });
   }),
 
-  number: computed('displayPriority', 'priority', function() {
-    const {
-      priority, displayPriority,
-    } = this;
-    if (!priority) {
-      return displayPriority;
-    }
-    return priority;
+  number: deprecatingAlias('priority', {
+    id: 'agendaitem-number-deprecated',
```

Comment:
Is er een reden waarom we de depricatingAlias willen gebruiken? Ik weet dat we in de stukken m.b.t. documenten er liever vanaf willen zijn dan ze er te houden.. Lijkt me dan niet handig om er hier wel bij te schrijven.

Reply:
[Deprecation](https://en.wikipedia.org/wiki/Deprecation)
> discouragement of use of some terminology, feature, design, or practice, typically because it has been superseded or is no longer considered efficient or safe, without completely removing it or prohibiting its use.
`number` is nog in gebruik, maar omdat het geen toegevoegde waarde heeft tegenover `priority`, is het gebruik ervan niet aangeraden. Bedoeling is inderdaad het uit te faseren. Ik zal een storytje aanmaken.

Reply:
Zie [KAS-1891](https://kanselarij.atlassian.net/browse/KAS-1891)

Reply:
In orde! Bedankt